project_closing/models/project.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Copyright 2013 Camptocamp Author Vincent Renaville
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import models, api
+
+
+class ProjectProject(models.Model):
+    _inherit = 'project.project'
+
+    @api.multi
+    def toggle_active(self):
+        if not self.env.context.get('doing_project_toggle_active'):
+            # When called directly from Project, delegate to Analytic Account
+            res = self.analytic_account_id.toggle_active()
```

Comment:
So I've got this error on archive a project 

![captura de 2017-05-29 18-31-28](https://cloud.githubusercontent.com/assets/4386443/26556836/3901123e-449d-11e7-8330-e49245f0403a.png)